native-netty-plot/README.md

```diff
@@ -10,30 +10,9 @@ This example further uses the [exp4j](https://www.objecthunter.net/exp4j/) libra
 
 ## Preparation
 
-To set up your development environment, you first need to [download](http://www.graalvm.org/downloads/) and set up GraalVM.
-GraalVM Enterprise Edition is preferable for the purpose of this example.
-
-1. [Download GraalVM](https://www.graalvm.org/downloads/), unzip the archive, export the GraalVM home directory as the `$JAVA_HOME` and add `$JAVA_HOME/bin` to the `PATH` environment variable.
-
-  On Linux:
-  ```bash
-  export JAVA_HOME=/home/${current_user}/path/to/graalvm
-  export PATH=$JAVA_HOME/bin:$PATH
-  ```
-  On macOS:
-  ```bash
-  export JAVA_HOME=/Users/${current_user}/path/to/graalvm/Contents/Home
-  export PATH=$JAVA_HOME/bin:$PATH
-  ```
-  On Windows:
-  ```bash
-  setx /M JAVA_HOME "C:\Progra~1\Java\<graalvm>"
-  setx /M PATH "C:\Progra~1\Java\<graalvm>\bin;%PATH%"
-  ```
-
-2. Install [Native Image](https://www.graalvm.org/docs/reference-manual/native-image/#install-native-image) by running:
+1. Download and install the latest GraalVM JDK with Native Image using the [GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader):
   ```bash
-  gu install native-image
+  bash <(curl -sL https://get.graalvm.org/jdk)
   ```
 
```

Comment:
Add the second step to clone the demos repository and navigate to this folder (as usual).